### PR TITLE
Icebox ruins expansion!

### PR DESCRIPTION
**Pull request by ZAMODA**
## About The Pull Request
Spices up icebox wastes with a bunch of different ruins! (only one right now, rest are in progress)
## Why It's Good For The Game
Icebox wastes as it is right now has few ruins and even fewer ghost role spawners, this aims to change that!
### Plasmaman Survivor ruin
Adds a ruin with 6 plasmamen sleepers, will be somewhat self sustaining. Meant to be a stand in for the golems and was inspired by charlie/old station and the lavaland syndicates
image
In progress image
## Changelog
:cl: Z Man Wit Z Plan
add: Adds more ruins to icebox!
add: Adds plasmamen ghostroles to icebox wastes.
code: Adds defines for plasmamen friendly atmos.
/:cl:

### CODE REVIEW

File: code/modules/mob_spawn/ghost_roles/mining_roles.dm [hunk 1]
```diff
@@ -323,3 +323,36 @@
...
+	icon_state = "psykerpod"
+	density = TRUE
+	mob_species = /datum/species/plasma
+	you_are_text = "You are a plasmaman, a sentient colony of fungi created by one of the plasma pits outside."
+	flavour_text = "Dispite not knowing your past or who you once were, you were able to keep information even through your rebirth \
+	The language you remember feels forign on your tongue, yet you still speak it."
+
+/obj/effect/mob_spawn/ghost_role/human/plasmaman/Initialize(mapload)
+	. = ..()
+	mob_name = pick("Hydrogen", "Helium", "Lithium", "Beryllium", "Boron", "Carbon", "Nitrogen", "Oxygen", \
```
Comment (MrMelbert) [inline code/modules/mob_spawn/ghost_roles/mining_roles.dm:341]: Rather than have this massive `pick`, just do `mob_name = pick(GLOB.plasmaman_names)`

**Comment by atlasle** [comment]
screenshots needed

**Comment by Singul0** [comment]
You should probably give this a proper name and description as soon as it's done

**Comment by ZAMODA** [comment]
> screenshots needed

Will add screenshots as soon as a ruin is completed!

**Comment by Xkallubar** [comment]
HELL YES, ICEBOX GHOST ROLES

File: code/modules/mob_spawn/ghost_roles/mining_roles.dm [hunk 2]
```diff
@@ -323,3 +323,23 @@
...
+/obj/effect/mob_spawn/ghost_role/human/plasmaman
+	name = "plasma filled pod"
+	desc = "A sleeper retrofitted to use plasma instead of oxygen."
+	prompt_name = "plasmaman survivor"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "psykerpod"
+	density = TRUE
+	mob_species = /datum/species/plasmaman
+	you_are_text = "You are a plasmaman, a sentient colony of fungi created by one of the plasma pits outside."
+	flavour_text = "Dispite not knowing your past or who you once were, you were able to keep information even through your rebirth \
```
Comment (ATH1909) [inline code/modules/mob_spawn/ghost_roles/mining_roles.dm:336]:
```suggestion
	flavour_text = "Despite not knowing your past or who you once were, you were able to keep information even through your rebirth \
```

File: code/modules/mob_spawn/ghost_roles/mining_roles.dm [hunk 3]
```diff
@@ -323,3 +323,23 @@
...
+	The language you remember feels forign on your tongue, yet you still speak it."
```
Comment (ATH1909) [inline code/modules/mob_spawn/ghost_roles/mining_roles.dm:337]:
```suggestion
	The language you remember feels foreign on your tongue, yet you still speak it."
```

File: code/modules/mob_spawn/ghost_roles/mining_roles.dm (hunk 2 above)
Comment (ATH1909) [reply code/modules/mob_spawn/ghost_roles/mining_roles.dm:336]: also, for the record, players who are converted into plasmamen via plasma rivers retain their memories/are still the same person

Comment (ZAMODA) [reply code/modules/mob_spawn/ghost_roles/mining_roles.dm:336]: > also, for the record, players who are converted into plasmamen via plasma rivers retain their memories/are still the same person
> 
> mechanically, anyway, idk what our lore on it is

Ah, alright! I will update the description accordingly!

**Comment by ZAMODA** [comment]
Compile maps is just not liking me very much.

**Comment by CVSthePharmacy** [comment]
can they PLEASE get a golem creation disk for their autolathe

**Comment by Admiral-Gigaton** [comment]
RIP icebox ghost roles PR, you will be missed dearly

**Comment by CVSthePharmacy** [comment]
can a maint reopen this?

**Pull request closed without merging**